otherlibs/win32graph/draw.c

```diff
@@ -116,10 +116,10 @@ CAMLprim value caml_gr_draw_text(value text,value x)
         SetTextAlign(grwindow.gcBitmap, TA_UPDATECP|TA_BOTTOM);
         SetTextAlign(grwindow.gc, TA_UPDATECP|TA_BOTTOM);
         if (grremember_mode) {
-                TextOut(grwindow.gcBitmap,0,0,(char *)text,x);
+                TextOutA(grwindow.gcBitmap,0,0,String_val(text),x);
```

Comment:
If you send UTF-8, its bytes will be intepreted in the current codepage (say, Latin-1) so that if your string has non-7 bit characters, garbage will be displayed.  This is the behaviour in trunk today.

I did not adapt `win32graph` simply to keep the diff as small as possible.  But it would be easy to do.  But maybe we could do it in a different PR, especially if it is going to be split off into a different library.